test/WebSites/BasicWebSite/_bower.readme

```diff
@@ -0,0 +1,7 @@
+To recreate minified JavaScript file,
```

Comment:
This file contains a few more details than the [`_grunt.readme`](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.TagHelpers/_grunt.readme) file in `src` and so I've used some markdown. Should I move this content to `readme.md` or rename the file to `_bower.readme.md`?